### ISSUE TITLE
Wrong implementation of MyConvo2d

### ISSUE THREAD

**Issue by ASDFJKLe** [jalola/improved-wgan-pytorch#2]
In your wgan.py file you have a class called MyConvo2d which takes stride as a parameter but when you set the stride of pytroch's nn.Conv2d you always set the stride to 1 the this parameter is kind of useless or wrong. I think in your model dir in the wgan.py file line 37 should look like this         
`self.conv = nn.Conv2d(input_dim, output_dim, kernel_size, stride=stride, padding=self.padding, bias = bias)`
instead of this 
`self.conv = nn.Conv2d(input_dim, output_dim, kernel_size, stride=1, padding=self.padding, bias = bias)`
It's probably important because when you call this class like you did in line 232 `self.conv2 = MyConvo2d(self.dim, 2*self.dim, 5,stride=2)` the stride never will be 2.

**Comment by jalola** [comment]
Thanks for pointing it. We actually intended to use `stride=1` for all conv2d in the model. 

Line 232 belongs `DCGANDiscriminator` and we actually don't use DCGANDiscriminator and FCGenerator for training. They are obsolete parts of our implementation. We should have removed them.

I've committed an update to clean them out.
